assets/components/paymentButtons/stripePopUpButton/stripePopUpButton.scss

```diff
@@ -6,13 +6,21 @@
   cursor: pointer;
   display: block;
   font-family: $gu-sans-web;
-  font-size: 14px;
+  font-size: 16px;
   height: $cta-height;
   padding-left: $gu-h-spacing;
   position: relative;
   text-align: left;
   width: 100%;
 
+
+  .svg-arrow-right-straight {
+    height: 16px;
+    position: absolute;
+    left: calc(100% - 34px);
```

Comment:
could you do `right: 34px` here?

Reply:
IE can be temperamental with `right` sometimes 😣